src/Microsoft.ML.FastTree/RegressionTree.cs

```diff
@@ -57,7 +57,7 @@ public abstract class RegressionTreeBase
         /// (2) the categorical features indexed by <see cref="GetCategoricalCategoricalSplitFeatureRangeAt(int)"/>'s
         /// returned value with nodeIndex=i is NOT a sub-set of <see cref="GetCategoricalSplitFeaturesAt(int)"/> with
         /// nodeIndex=i.
-        /// Note that the case (1) happens only when <see cref="CategoricalSplitFlags"/>[i] is true and otherwise (2)
+        /// Note that the case (1) happens only when <see cref="CategoricalSplitFlags"/>[i] is false and otherwise (2)
```

Comment:
It looks to me this doc was wrong, as it's inconsistent with what these other lines below say:

https://github.com/dotnet/machinelearning/blob/8660ecc7742a02ac896dc77c93c67c366ff4647e/src/Microsoft.ML.FastTree/RegressionTree.cs#L76-L80

https://github.com/dotnet/machinelearning/blob/8660ecc7742a02ac896dc77c93c67c366ff4647e/src/Microsoft.ML.FastTree/RegressionTree.cs#L100-L106